classic/src/view/container/Viewport.js

```diff
@@ -1,6 +1,12 @@
 Ext.define('momo.view.container.Viewport', {
     extend: 'Ext.container.Viewport',
 
+    requires: [
+               'BasiGX.*',
+               'GeoExt.*',
+               'momo.*'
```

Comment:
Yeah, we discussed this a lot of times. It's definitely not perfect, but I didn't see any other way as we declare our application context (and thereby the classes to load) dynamically.